<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialogue</class>
 <widget class="QDialog" name="AddItemDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="itemNamelabel">
     <property name="text">
      <string>Item Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="itemNameEdit">
     <property name="placeholderText">
      <string>Item Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="itemDescLabel">
     <property name="text">
      <string>Item Description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="itemDescEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Description...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="itemPriceLabel">
     <property name="text">
      <string>Item Price</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="itemPriceEdit">
     <property name="prefix">
      <string>Â£</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="itemImageUriLabel">
     <property name="text">
      <string>Item Image</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="itemImageUriEdit">
     <property name="placeholderText">
      <string>/cdn/default_image.jpg</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QGraphicsView" name="itemImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="changeImage">
     <property name="text">
      <string>Change Image</string>
     </property>
     <property name="icon">
      <iconset theme="folder_open"/>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="itemActiveEdit">
     <property name="text">
      <string>Start Activated</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="itemTypeEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="itemTypeEditLabel">
     <property name="text">
      <string>Item Type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="itemPrepTimeEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="itemPrepTimeLabel">
     <property name="text">
      <string>Prep Time</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddItemDialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
